pysr/sr.py

```diff
@@ -1989,11 +2004,70 @@ def get_hof(self):
             return ret_outputs
         return ret_outputs[0]
 
+    def plot(self, ax, threshold=0.05, fontsize=10):
+        """Create a plot of the best equations.
+
+        Parameters
+         ----------
+        threshold : float
```

Comment:
These are out of order. Also, `fontsize` is missing.

More generally, it would be good to add `**kwargs` to the plotting function to tweak the other settings.